src/System.Management.Automation/engine/PSVersionInfo.cs

```diff
@@ -5,6 +5,7 @@
 using System.Diagnostics;
 using System.Reflection;
 using System.Collections;
+using System.Collections.Generic;
```

Comment:
Once you make `PSVersionTableComparer` implements `IComparer` instead of deriving from `Comparer<object>`, this using directive can be removed.